fetch/cosmwasm_contract/src/contract.rs

```diff
@@ -659,7 +776,7 @@ fn verify_refund_swap_id<S: Storage>(id: u64, storage: &S) -> HandleResult {
  * ***************************************************/
 
 fn only_admin<S: Storage>(env: &Env, storage: &S) -> HandleResult {
-    _only_role(&AccessRole::Admin, env, storage).or(_only_original_owner(env, storage))
+    _only_role(&AccessRole::Admin, env, storage) //.or(_only_original_owner(env, storage))
```

Comment:
you can ditch commented-out code.

Reply:
sure, I kept it because I was not sure about this